### PR TITLE
[Components with more than 1 service] Refactor Saml components from saml modules

### PR DESCRIPTION
**Pull request by GabrielHenriquedev**

### CODE REVIEW

**Comment by GabrielHenriquedev** [comment]
ci:test:security

**Comment by GabrielHenriquedev** [comment]
ci:test:saml

**Comment by Kvale1733** [comment]
ci:test:saml

**Comment by catarineaguiar** [comment]
@GabrielHenriquedev semver error:

```
     [exec] com.liferay.semantic.versioning.SemanticVersioningTest > testSemanticVersioning[/dxp/apps/saml/saml-api] FAILED
     [exec]     java.lang.AssertionError:   PACKAGE_NAME                                       DELTA      CUR_VER    BASE_VER   REC_VER    WARNINGS  
     [exec]     = ================================================== ========== ========== ========== ========== ==========
     [exec]     * com.liferay.saml.util                              MAJOR      2.2.0      2.2.0      3.0.0      VERSION INCREASE REQUIRED
     [exec]     	+   interface  com.liferay.saml.util.SamlHttpRequest
     [exec]     		+   access     abstract
     [exec]     		+   method     getEntityDescriptorString(javax.servlet.http.HttpServletRequest)
     [exec]     			+   access     abstract
     [exec]     			+   return     java.lang.String
     [exec]     		+   method     getRequestPath(javax.servlet.http.HttpServletRequest)
     [exec]     			+   access     abstract
     [exec]     			+   return     java.lang.String
     [exec]     	-   interface  com.liferay.saml.util.SamlHttpRequestUtil
     [exec]     		-   access     abstract
     [exec]     		-   method     getEntityDescriptorString(javax.servlet.http.HttpServletRequest)
     [exec]     			-   access     abstract
     [exec]     			-   return     java.lang.String
     [exec]     		-   method     getRequestPath(javax.servlet.http.HttpServletRequest)
     [exec]     			-   access     abstract
     [exec]     			-   return     java.lang.String
     [exec]     [Baseline Warning] Bundle Version Change Recommended: 8.0.0
```
full log [here](https://testray.liferay.com/reports/production/logs/2023-07/test-1-25/test-portal-acceptance-pullrequest(master)/1843/modules-semantic-versioning-jdk8/0/0/jenkins-console.txt.gz?authuser=0)

File: modules/dxp/apps/saml/saml-api/src/main/java/com/liferay/saml/util/SamlHttpRequest.java [hunk 1]
```diff
@@ -21,7 +21,7 @@
 /**
  * @author Michael C. Han
  */
-public interface SamlHttpRequestUtil {
+public interface SamlHttpRequest {
```
Comment (dantewang) [inline modules/dxp/apps/saml/saml-api/src/main/java/com/liferay/saml/util/SamlHttpRequest.java:24]: @GabrielHenriquedev `SamlHttpRequest` looks like a class representing an HTTP request; a better name could be `SamlHttpRequestHelper`.

File: modules/dxp/apps/saml/saml-opensaml-integration/src/test/java/com/liferay/saml/opensaml/integration/internal/metadata/MetadataManagerImplTest.java [hunk 2]
```diff
@@ -51,7 +51,7 @@ public void testGetRequestPath() {
 
 		Assert.assertEquals(
 			"/c/portal/login",
-			_samlHttpRequestUtilImpl.getRequestPath(mockHttpServletRequest));
+			_samlHttpRequestImpl.getRequestPath(mockHttpServletRequest));
```
Comment (dantewang) [inline modules/dxp/apps/saml/saml-opensaml-integration/src/test/java/com/liferay/saml/opensaml/integration/internal/metadata/MetadataManagerImplTest.java:54]: @GabrielHenriquedev It seems this test only tests the extracted class, please rename it.

**Review by dantewang — CHANGES_REQUESTED** [review]
@GabrielHenriquedev Please see my comment and update the renaming, thanks!

**Comment by GabrielHenriquedev** [comment]
ci:test:sf

**Comment by GabrielHenriquedev** [comment]
ci:test:relevant

**Comment by dantewang** [comment]
@zhangyan0701 Could you just double check the relevant result? I believe the new push only contains some renaming, so no real logic is changed.

**Comment by zhangyan0701** [comment]
No regressions found.

**Comment by dantewang** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/137796

**Pull request closed without merging**